tests/integration/common/s2n_test_reporting.py

```diff
@@ -40,23 +45,32 @@ class Result:
 
     """
 
-    def __init__(self, error=None):
-        self.error = error
-        self.status = Status.PASSED if error == None else Status.FAILED
+    def __init__(self, error_msg=None):
+        self.error_msg = error_msg
+        self.client_error = None
+        self.server_error = None
+        self.status = Status.PASSED if error_msg == None else Status.FAILED
```

Comment:
is it more common to use `error_msg is None`?  i forget my python best practices.